docs/source/cli_help/status.rst

```diff
@@ -80,28 +81,7 @@ With ``--color=always``, git machete always emits colors and with ``--color=auto
 
 --no-detect-squash-merges         Only consider *strict* (fast-forward or 2-parent) merges, rather than rebase/squash merges, when detecting if a branch is merged into its upstream (parent).
 
- .. note::
-
-    To make it easier to select branch name from the ``status`` output on certain terminals
-    (e.g. `Alacritty <https://github.com/alacritty/alacritty>`_), you can add an extra
```

Comment:
Pls open a task for automagically selecting line lengths (might be non-trivial tho, not a high priority definitely)

Reply:
https://github.com/VirtusLab/git-machete/issues/652